doc/hotfix-process.md

```diff
@@ -0,0 +1,63 @@
+Hotfix Release Process
+======================
+
+Hotfix releases are versioned by incrementing the build number of the latest
+release. For example:
+
+    First hotfix:  v1.0.11   -> v1.0.11-1
+    Second hotfix: v1.0.11-1 -> v1.0.11-2
+
+In the commands below, <RELEASE> and <RELEASE_PREV> are prefixed with a v, ie.
+v1.0.11 (not 1.0.11).
+
+## Create a hotfix branch
+
+Create a hotfix branch from the previous release tag, and push it to the main
+repository:
+
+    $ git branch hotfix-<RELEASE> <RELEASE_PREV>
+    $ git push 'git@github.com:zcash/zcash' hotfix-<RELEASE>
+
+## Merge hotfix PRs
+
+Developer should create hotfix PRs based on the hotfix branch. Each PR should be
+reviewed as normal, and then the following process should be used to merge:
+
+- A CI merge build is manually run, by force-triggering the pr-merge builder
+  with the following fields set:
+
+  - Repository: https://github.com/<DevUser>/zcash
+    - <DevUser> must be in the set of "safe" users as-specified in the CI
+      config.
+  - Branch: name of the hotfix PR branch (not the hotfix release branch).
+
+- A link to the build and its result is manually added to the PR as a comment.
+
+- If the build was successful, the PR is merged via the GitHub button.
```

Comment:
What happens if the developer accidentally hits the GitHub button to merge? Can the button be enabled for clicking only on a successful build?  Add instructions on how to revert the merge?

Reply:
Same problem as with any regular merge - someone could accidentally hit the GitHub merge button without going through Homu. There are ways to enable only after N reviews, but no way (that I am aware of) to enable after some automated review. I agree that there is a (slightly) higher chance of it occurring here though, give that the merge button is involved in the regular process; hopefully in future Homu could be extended (or duplicated) to support the hotfix model.